pypeit/specobj.py

```diff
@@ -138,6 +138,7 @@ class SpecObj(datamodel.DataContainer):
                  'BOX_CHI2': dict(otype=np.ndarray, atype=float,
                                   descr='Reduced chi2 of the model fit for this spectral pixel'),
                  'BOX_RADIUS': dict(otype=float, descr='Size of boxcar radius (pixels)'),
+                 'SPAT_BOX_RADIUS': dict(otype=float, descr='Size of boxcar radius (arcsec)'),
```

Comment:
I understand that it matches the convention of `SPAT_FWHM`, but I don't think the combination of `BOX_RADIUS` and `SPAT_BOX_RADIUS` make it clear enough that one is in pixels and the other in arcsec.  At the risk of having to make a lot of edits, could we change this to something like `BOX_R_PIX` and `BOX_R_ASEC`?

Reply:
Done.